migration/v0.30/apply/90-psp-violations.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+declare -a results
```

Comment:
Not sure what I feel about global variables, but as it is so small script I guess it is fine.

Reply:
Agree